app/models/resource.rb

```diff
@@ -8,14 +8,22 @@ class Resource < ActiveRecord::Base
 
   # Validations
   validates_presence_of :title, :paradigm_id
+  validates_numericality_of :paradigm_id, only_integer: true
 
   # Scopes
   scope :alphabetical, -> { order("title") }
   scope :active, -> { where('active = ?', true) }
   scope :inactive, -> { where('active = ?', false) }
 
-
   # Methods
+
+  def full_link
+    if !self.link.include?("http://")
+      self.link.insert(0,"http://")
```

Comment:
also we probably shouldn't modify the link itself in this. instead what we can do is probably use a before_save so that we can gaurantee that the http or https is included before we save it or add it if its not included. This is similar to the reformat_phone method that we did for 272